app/views/settings/edit.html.erb

```diff
@@ -2,20 +2,25 @@
 
 <div class="t-body">
   <h2 class="page__subheading"><%= t ".mfa.multifactor_auth" %></h2>
-  <% if @user.mfa_enabled? %>
-    <h2><%= t(".mfa.otp")%></h2>
-    <p><%= t(".mfa.enabled_html") %></p>
-    <%= form_tag multifactor_auth_path, method: :put, id: "mfa-edit" do %>
-      <%= label_tag :level, t(".mfa.level.title"), class: "form__label" %>
-
+  <% if @user.totp_enabled? || @user.webauthn_enabled?%>
```

Comment:
This makes more sense as `@user.mfa_enabled?` imo since we are relying on the mfa_level as the source of truth.

That being said, for users that only have webauthn enabled currently, their MFA level is `disabled`. When I test this branch, it shows that the MFA level is UI and API (but in the db it's disabled).

<img width="882" alt="Screenshot 2023-06-02 at 2 09 01 PM" src="https://github.com/rubygems/rubygems.org/assets/42748004/3b249d1a-4b64-4146-9514-d337046d2082">

The level that these users should be now is `ui_and_gem_signin` because if you only have webauthn currently, you get prompted to MFA only on signin.

To help solve this inconsistency, we can add a clause to `mfa_enabled` to update the user's mfa level if they only have webauthn enabled.

```
  def mfa_enabled?
    webauthn_credentials.present? && mfa_disabled?
      self.mfa_level = "ui_and_gem_signin"
      save!(validate: false)
    end

    !mfa_disabled?
  end
```
Then, we should migrate all of the rest of users to `ui_and_gem_signin` using a task and remove the clause.

Reply:
Added the update in https://github.com/rubygems/rubygems.org/pull/3821/commits/102b61a771749ca5c39dc0c7b26d47aeb528dd6f

> Then, we should migrate all of the rest of users to ui_and_gem_signin using a task and remove the clause.
Hasn't been done yet